### PR TITLE
Update transactions object

### PR DESCRIPTION
**Pull request by rplopes**
- replace basic example with no currency exchange and no fees with one which includes them, for completeness sake
- Add new fields `application`, `network`, `priority`, `reference`, `fees.target`, `params.rate`, `params.type`, `normalized.fee`, `normalized.target`, `destination.isMember`, `destination.node`, `origin.isMember` and `origin.node`
- Fix inconsistencies in descriptions of nested objects

### CODE REVIEW

File: _entities.md [hunk 1]
```diff
@@ -179,17 +210,21 @@ Transactions record the movement of value into, within and out of the Uphold net
...
 Property     | Description
 ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+application  | The application that created the transaction.
 id           | A unique ID on the Uphold Network associated with the transaction.
 type         | The nature of the transaction. Possible values are `deposit`, `transfer` and `withdrawal`.
 message      | A message or note provided by the user at the time the transaction was initiated, with the intent of communicating additional information and context about the nature/purpose of the transaction.
-denomination | The funds to be transferred, as originally requested. See "Denomination" below.
-fees         | The fees that were applied to the transaction.
+denomination | The funds to be transferred, as originally requested. See [Denomination](#denomination).
+fees         | The fees that were applied to the transaction. See [Fees](#fees).
```
Comment (waldyrious) [inline _entities.md]: 👍

File: _entities.md [hunk 2]
```diff
@@ -242,7 +280,9 @@ Property   | Description
 amount     | The amount to be transacted.
 commission | The total commission taken on this transaction, either at origin or at destination.
 currency   | The currency in which the amount and commission are expressed. The value is always `USD`.
+fee        | The normalized fee amount.
 rate       | The exchange rate for this pair.
+target     | Can be `origin` or `destination` and determines where the fee was applied.
```
Comment (waldyrious) [inline _entities.md]: Looks like all other entries start with "The foo". This should probably adopt the same format, for consistency.

File: _entities.md [hunk 3]
```diff
@@ -256,6 +296,8 @@ commission  | The commission charged by Uphold to process the transaction.
 currency    | The currency of the funds at the origin.
 description | The name of the sender.
 fee         | The Bitcoin network Fee, if origin is in BTC but destination is not, or is a non-Uphold Bitcoin Address.
+isMember    | A boolean signaling if the origin user has completed the membership process.
+node        | Details about the transaction origin node.
```
Comment (waldyrious) [inline _entities.md]: Same as above: these two would fit better if they started with "The foo".

File: _entities.md [hunk 4]
```diff
@@ -277,6 +319,8 @@ commission  | The commission charged by Uphold to process the transaction. Commi
 currency    | The denomination of the funds at the time they were sent/received.
 description | The name of the recipient. In the case where money is sent via email, the description will contain the email address of the recipient.
 fee         | The Bitcoin network Fee, if destination is a BTC address but origin is not.
+isMember    | A boolean signaling if the destination user has completed the membership process.
+node        | Details about the transaction destination node.
```
Comment (waldyrious) [inline _entities.md]: Same as above: these two would fit better if they started with "The foo".

File: _entities.md (hunk 2 above)
Comment (rplopes) [reply _entities.md]: This was extracted from the `### Fees` section, which was already using this description. Maybe we can improve some of these descriptions in the future.

**Comment by waldyrious** [comment]
@rplopes sorry, I can't figure out what changed in your new commit :/ Can you clarify?

File: _entities.md (hunk 2 above)
Comment (rplopes) [reply _entities.md]: Sure. It would be nice to have a list of low-hanging fruit like this to improve in the near future.

**Comment by rplopes** [comment]
@waldyrious only the `node` description:

```diff
❯ git diff 5175c72 2359cf1
diff --git a/_entities.md b/_entities.md
index 3f83096..8e69ded 100644
--- a/_entities.md
+++ b/_entities.md
@@ -297,7 +297,7 @@ currency    | The currency of the funds at the origin.
 description | The name of the sender.
 fee         | The Bitcoin network Fee, if origin is in BTC but destination is not, or is a non-Uphold Bitcoin Address.
 isMember    | A boolean signaling if the origin user has completed the membership process.
-node        | Details about the transaction origin node.
+node        | The details about the transaction origin node.
 rate        | The rate for conversion between origin and destination, as expressed in the currency at origin (the inverse of `destination.rate`).
 sources     | The transactions where the value was originated from (id and amount).
 type        | The type of endpoint. Possible values are 'card' and 'external'.
@@ -320,7 +320,7 @@ currency    | The denomination of the funds at the time they were sent/received.
 description | The name of the recipient. In the case where money is sent via email, the description will contain the email address of the recipient.
 fee         | The Bitcoin network Fee, if destination is a BTC address but origin is not.
 isMember    | A boolean signaling if the destination user has completed the membership process.
-node        | Details about the transaction destination node.
+node        | The details about the transaction destination node.
 rate        | The rate for conversion between origin and destination, as expressed in the currency at destination (the inverse of `origin.rate`).
 type        | The type of endpoint. Possible values are 'email', 'card' and 'external'.
```

File: _entities.md (hunk 2 above)
Comment (waldyrious) [reply _entities.md]: Great. I've created #160 to track this particular issue, and added a ["bite-size" label](https://github.com/uphold/docs/labels/bite-size) for similar issues. Let me know if you prefer a different name for the label, and please tag any relevant existing issues with it.

**Pull request merged**